requirements.txt

```diff
@@ -23,7 +23,7 @@ defusedxml==0.5.0
 docopt==0.6.2
 docutils==0.13.1
 -e git+https://github.com/AusDTO/dto-digitalmarketplace-apiclient.git@7e145229335408ed0522577f8e4b521a3e399a04#egg=dto_digitalmarketplace_apiclient==whatever
--e git+https://github.com/AusDTO/dto-digitalmarketplace-utils.git@640f9af164a1555d274a16d2aa47d4f31b85b6cc#egg=dto_digitalmarketplace_utils==whatever
+-e git+https://github.com/AusDTO/dto-digitalmarketplace-utils.git@5ae96ede136c99ec1b89a5eb80c623b4f74983e4#egg=dto_digitalmarketplace_utils==whatever
```

Comment:
This is updating to the dm-utils commit which introduced automatic upload filename renaming to avoid overwriting.